pnpm-lock.yaml

```diff
@@ -17029,6 +17056,49 @@ packages:
       - yaml
     dev: true
 
+  /tsup@8.3.5(tsx@4.19.2)(typescript@5.5.4):
```

Comment:
^ this caught my eye. We can probably just run `pnpm dedeup` in its own PR to clean all of this up.